fs-api/src-unix/System/IO/FS.hs

```diff
@@ -1,6 +1,6 @@
 {-# LANGUAGE PackageImports #-}
 
-module Ouroboros.Consensus.Storage.IO (
+module System.IO.FS (
```

Comment:
We should also add some comments to this module as well (as a separate PR is fine).

Reply:
Idem https://github.com/input-output-hk/ouroboros-network/pull/4425#discussion_r1132383639